src/gui/rss/feedlistwidget.cpp

```diff
@@ -85,7 +85,7 @@ void FeedListWidget::handleFeedStateChanged(RSS::Feed *feed)
 
     QIcon icon;
     if (feed->isLoading())
-        icon = QIcon(QStringLiteral(":/icons/loading.png"));
+        icon = UIThemeManager::instance()->getIcon(QStringLiteral("loading"));
```

Comment:
here too

Reply:
this is to match the surrounding code.

Reply:
I guess replacing with QStringLiteral makes more sense.

Reply:
>I guess replacing with QLatin1String makes more sense.

Indeed. As for other `QStringLiteral` usages, I will handle them later.

Reply:
QLatin1String makes sense when passed to function that has an overload taking QLatin1String as parameter. When you need to construct QString QSringLiteral is better choice since it makes less conversions (it allocates string data at compile time).
The exception case can be the string that is very rarely used or used once in some initialization code.